File: ConfigurePcLaravel/configurePc-App/app/Models/Order.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class Order extends Model
{
      protected $fillable = [
        'order_date',
        'status',
        'customer_id',
    ];

    public function customer()
    {
        return $this->belongsTo(Customer::class, 'customer_id', 'id');
    }

    public function computer()
    {
        return $this->hasOne(Computer::class, 'order_id', 'id');
    }
}

File: ProductionSystem/Persistence.Plugin/Models/order.cs
using System;
using System.Collections.Generic;

namespace PersistencePlugin.Models;

public partial class order
{
    public long id { get; set; }

    public DateOnly order_date { get; set; }

    public string status { get; set; } = null!;

    public long customer_id { get; set; }

    public DateTime? created_at { get; set; }

    public DateTime? updated_at { get; set; }

    public virtual computer? computer { get; set; }

    public virtual customer customer { get; set; } = null!;
}
